cctbx/sgtbx/direct_space_asu/check_redundancies.py

```diff
@@ -237,12 +235,16 @@ def grid_asu(
   return result
 
 def compare_redundancies(a, b):
-  return cmp(len(b[1]), len(a[1]))
+  # Deprecated. Do not use
+  if len(b[1]) < len(a[1]):
+    return -1
+  elif len(b[1]) > len(a[1]):
+    return 1
+  else:
+    return 0
```

Comment:
Duplicate `cmp`
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.